app/services/client.ts

```diff
@@ -17,27 +18,30 @@ const FetchApi = async ({
   const body =
     contentType === "multipart/form-data" ? payload : JSON.stringify(payload);
   apiOverride = isAuth ? ApiConfig.BASE_URL_AUTH : ApiConfig.BASE_URL_API;
+  apiOverride = isGtrackit ? ApiConfig.GTRACKIT_BASE_URL_API : apiOverride;
```

Comment:
isGtrackit is not needed as the URL should be decided based on login user's region